### ISSUE TITLE
Professions with faulty bionics have not been adjusted, stat, point, and gear-wise, to reflect the new changes

### ISSUE THREAD

**Issue by Xhuis** [CleverRaven/Cataclysm-DDA#24781]
Note: An exception may be made with Cyberjunkie, as they have Sensory Dulling and are not arbitrarily resource-locked.

The recent bionics overhaul appears to have stabilized by now, and the discussions are no longer a flaming pyre, but more of a smoldering ash pile as people talk about it. I was suggested to take my points into an issue, so here's that.

The short version is in the title; professions with faulty bionics (including Cyberjunkie, Failed Cyborg, Broken Cyborg, etc.) haven't been updated at all to reflect the sudden new bionics changes. The net result is that some have items with no apparent purpose, and, most commonly, the starts are ludicrously difficult for the points they afford.

Broken Cyborg is the worst of the lot - you need *eighteen* anesthetic kits to fully remove your malfunctioning bionics. *Eighteen.* On top of that, you're not guaranteed to actually succeed in removing it, meaning you'll likely need more than that unless you're exceedingly lucky. Broken Cyborg is no longer a question of rushing to remove your faulty bionics, but an acceptance of highly-debilitating long-term drawbacks, and the measly 2 points it offers don't reflect this.

List of things, in no particular order, that are currently problematic with the starts I can find:
* Broken Cyborg only awards 2 points and starts with 18 faulty bionics that they can't remove without finding six anesthetic kit spawns at the very least
* Cyberjunkie still spawns with a first aid kit and pocket knife, which they do not have the room to carry and so the kit immediately falls to the ground upon moving; removal isn't as difficult, and has the skills based around old bionics (electronics, first aid, mechanics) rather than the new ones
* Bionic Monster costs 2 points and starts with 6 faulty bionics that they can't remove without finding two anesthetic kit spawns at the very least

If the point values are to be kept, the best solution to me is to outfit them with the Sensory Dulling CBM, so that they're able to remove their bionics at an Autodoc without also having to scavenge for a great deal of anesthesia.

If this is too unbalancing, then ideas that come to mind are to remove the anesthesia requirements for removing faulty bionics, adding a new bionic that simply disables needing anesthesia to uninstall, allowing the player to suffer a tremendous amount of pain and proceed without anesthesia if they have enough depressants in their system to put them to sleep as the Autodoc begin, Finally, there's no problem with simply buffing the point values, as well.

**Comment by DracoGriffin** [comment]
You really should approach this from an encompassing view, because otherwise, comparing previous point cost of those professions/scenarios to now isn't as significant as all the recent changes lately that affect all professions/scenarios. In order to give a fair view and rebalancing for the professions you've outlined, so too the rest need to be reviewed as they all have been quite impacted by the recent changes.

A user on C:DDA subreddit made a [pretty decent and recent review of a lot of the professions](https://www.reddit.com/r/cataclysmdda/comments/962nfm/my_list_of_the_best_default_classes_which/) (not all-encompassing as his approach was min/maxing, but nonetheless) and gives a [thoughtful analysis here](https://www.reddit.com/r/cataclysmdda/comments/962nfm/my_list_of_the_best_default_classes_which/e3yw13m/).

So there are a few approaches to this issue, but I have yet to see one (although not to see there isn't a thought-out one in the many closed issues over the years) that approaches this tactfully and from a more... "neutral" stance.

One way I have been tossing around is instead of purely going off of stat/trait/skill points, but giving values to some of the afflictions --- like what should starting "infected" cost, especially now with no longer having access to first aid kids to remove infection as well as previous infection on/off toggle has now become a languorous process to endure. This way, when some of these starts are applied, it is more because "well X costs -A, but Y gives +B" rather than "Well, we think this should cost this because of Z". This would also potentially involve re-evaluating the costs/points associated with traits/skills/stats to be more in-line but that involves a lot more discussion and brainstorming.

In conclusion, I'd really hazard against approaching this as "I like these professions and playstyles, so it should be adjusted to my liking" and more as "Hey guys, a lot of these professions and scenarios are completely out of date/context compared to the current changes and system that has developed over the years, how should we approach this?"

**Comment by Night-Pryanik** [comment]
> remove the anesthesia requirements for removing faulty bionics

Just no.

> adding a new bionic that simply disables needing anesthesia to uninstall

Why? Sensory Dulling is already here, no need to duplicate the functional.

> allowing the player to suffer a tremendous amount of pain

That might be an interesting mechanic, but that would require a large thinking over as that will make all autodoc mechanics useless (make operation, suffer some pain (maybe even HUGE AMOUNTS of pain), then wait some time, as pain will fall down sooner or later, walk away happily).

> proceed without anesthesia if they have enough depressants in their system to put them to sleep as the Autodoc begin

Depressants strong enough to put you in sleep for the entire operation process are called anesthesia.

**Comment by Xhuis** [comment]
@draco I attempted to do so by outlining all of the ones affected by faulty bionics as I feel those are the most greatly outdated. I do not personally have a problem with most of the existing ones and think that they're most okay.

@night You made these bionic changes. What do you think should be done? And, your statement about depressants is not fully true - one can have enough depressants in their system to knock them out from meth withdrawal alone; it will constantly knock them out.

**Comment by Night-Pryanik** [comment]
We can always simply increase the points given to Broken Cyborg and Bionic Monster.

Cyberjunkie should receive some bag to hold his possessions, first aid kit and a pocket knife could be removed as they ain't needed anymore to uninstall bionics, and change the skills to reflect the changes. This should be the easiest changes.

Another thing that should enhance the game overall and could help with the profession problems you outlined is to change the "one charge of anesthetic per operation" mechanic to make autodoc put the patient in desired duration of narcosis. This will permit several installation or uninstallation operations (based on their difficulty) in a row. Say, one charge of the anesthetic kit could work for an hour, so one can install up to 9 Power Storage CBMs with only one anesthetic kit in a row, without exiting from the narcosis! Sadly, all faulty bionics require two hours to uninstall, but we can always tweak the duration of one anesthetic charge to counter that.

**Comment by Inglonias** [comment]
> We can always simply increase the points given to Broken Cyborg and Bionic Monster.
[That was tried.](https://github.com/CleverRaven/Cataclysm-DDA/pull/24592) [Kevin reverted the change.](https://github.com/CleverRaven/Cataclysm-DDA/pull/24592)

Unless you can convince the man upstairs that this is a viable option, you're going to need to think of something better.

**Comment by Night-Pryanik** [comment]
> That was tried. Kevin reverted the change.

Yes, I know that. I think that's because increase was too radical - three times from the previous, from -2 to -6. I'd say it should be -3 or -4 maximum.

**Comment by kevingranade** [comment]
> The recent bionics overhaul appears to have stabilized by now,

Craftable anesthesia kits are still in flux as well as additional spawn locations for kits.  
No one has taken up the *very* promising "non surgical CBM deactivation concept, which would also obviate problems with these starts.  
Batch CBM removal would completely change the resource usage surrounding this issue.  
I dont remember where, but someone suggested simply removing the leaky bionic, which is the only truly life-threatening faulty bionic AFAICT.

I fully expect most of these to be implemented, at which point this issue becomes moot.

> you need eighteen anesthetic kits to fully remove your malfunctioning bionics

You keep repeating this, but it is *not* making your case for you. Not every faulty bionic is a big deal, a number of them are more flavor than anything else.

> Cyberjunkie still spawns with a first aid kit and pocket knife, which they do not have the room to carry and so the kit immediately falls to the ground upon moving

That's an acceptable fix.

> outfit them with the Sensory Dulling CBM

This is a massive long-term buff and is not ok.

Regarding other suggestions:  
Situational anesthesia requirements have no rationale, surgery is surgery.  If an autodoc is involved in the deactivation process, and *that* didn't require anesthesia, that would be fine.  
A new bionic for uninstallation is simply nonsensical.  Why would anyone make such a thing instead of the sensory dulling CBM?  Why would they install it on experiment subjects?  

Anesthesia is non-optional, full stop.

The professions with faulty bionics are intended to be a challenge, pumping up the point value of them in order to encourage their use just leads them being an overly attractive target for min-maxers, which is ridiculous since *you can change how many points you have in chargen*.

**Comment by kevingranade** [comment]
Dead issue, these professions are intended to be challenges, and they're fine like they are.
Feel free to open these issues though:
* non surgical CBM deactivation
* Batch CBM removal
* remove the leaky bionic
* Fix junkie spawns (probably done already?)
